helm-charts/agentqna/README.md

```diff
@@ -1,15 +1,78 @@
 # AgentQnA
 
-Helm chart for deploying AgentQnA service.
+Helm chart for deploying AgentQnA example.
+
+See [AgentQnA](https://github.com/opea-project/GenAIExamples/tree/main/AgentQnA) for the example details.
+
+Note that this is an example to demonstrate how agent works and tested with prepared data and questions. Using different datasets, models and questions may get different results.
```

Comment:
I'm not sure how exactly to interpret this text.  It may be simpler like this:
```suggestion
Helm chart for deploying AgentQnA example. It demonstrates how agent works, using prepared data and questions. See [AgentQnA overview](https://github.com/opea-project/GenAIExamples/tree/main/AgentQnA#overview) for the details.

Using different datasets, models and questions may get different results.
```